app/models/organisation.rb

```diff
@@ -2,24 +2,24 @@
 require 'string'
 
 class Organisation < BaseOrganisation
-
+  has_many :events
```

Comment:
did @sstamoulas not also add a similar relation in his PR?

Reply:
@tansaku the change is not on develop at least https://github.com/AgileVentures/LocalSupport/blob/develop/app/models/organisation.rb that is why I added the relation :+1: